### PR TITLE
fix facet search

### PR DESCRIPTION
**Pull request by ThibodeauJF**

### CODE REVIEW

File: src/components/facet-search.tsx [hunk 1]
```diff
@@ -2,17 +2,15 @@ import {
...
-import {FunctionComponent, useState} from 'react';
+import {FunctionComponent} from 'react';
 
 export interface FacetSearchProps {
   controller: HeadlessFacetSearch;
   facetSearchState: FacetSearchState;
 }
 
 export const FacetSearch: FunctionComponent<FacetSearchProps> = (props) => {
-  const [searchBoxValue, setSearchBoxValue] = useState('');
```
Comment (ThibodeauJF) [inline src/components/facet-search.tsx:13]: don't need a local state, now the query's state is exposed from headless, that can be used instead

File: src/components/facet-search.tsx [hunk 2]
```diff
@@ -24,17 +22,16 @@ export const FacetSearch: FunctionComponent<FacetSearchProps> = (props) => {
...
         onInput={(e) => updateSearch(e.currentTarget.value)}
       />
-      {searchBoxValue !== '' && (
+      {props.facetSearchState.query !== '' && (
         <ul style={searchResultsStyle}>
           {props.facetSearchState.values.map((facetSearchValue) => (
             <li key={facetSearchValue.rawValue}>
               <button
                 onClick={() => {
                   props.controller.select(facetSearchValue);
```
Comment (ThibodeauJF) [inline src/components/facet-search.tsx:34]: select triggers an executeSearch, which clears the SearchBox query as well as the FacetSearchBox query

**Review by jpmarceau — APPROVED** [review]
That's great JF, thanks!

**Comment by ThibodeauJF** [comment]
You're welcome!

**Pull request merged**